src/pages/PassportRedemption/RedemptionClaimScreen.tsx

```diff
@@ -34,8 +34,7 @@ const PassportRedemptionClaim = ({ setCurrentScreenView }: Props) => {
       zip_code: '',
     },
     logo_url: '',
-    reward: '',
-    reward_detail: '',
```

Comment:
why delete?

Reply:
Ooh sorry! I think I misunderstood the wireframe. I'll add it back!